jsonnet/kube-prometheus/components/k8s-control-plane.libsonnet

```diff
@@ -7,6 +7,7 @@ local defaults = {
   commonLabels:: {
     'app.kubernetes.io/name': 'kube-prometheus',
     'app.kubernetes.io/part-of': 'kube-prometheus',
+    'app.kubernetes.io/component': 'prometheus',
```

Comment:
I'm not sure that 'prometheus' is the right component here since we're looking at control plane services.